build/toolchain/webos/webos_toolchain.gni

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  cc_webos = ""
+  cxx_webos = ""
+  ar_webos = ""
+  ld_webos = ""
+}
+
+import("//build/toolchain/gcc_toolchain.gni")
+
+template("webos_toolchain") {
+  print("WebOS_ToolChian")
+  gcc_toolchain(target_name) {
+    ar = ar_webos
+    cc = cc_webos + "  -O2 -Wno-format-security -Wno-error -Wno-cpp"
```

Comment:
Please don't specify warning or optimization flags in the toolchain as these are controlled by args.

Please don't build with -Wno-error

Reply:
Let me remove this and control further actions by args.
Thank you for your review.